src/test/java/teammates/test/cases/action/InstructorStudentRecordsPageActionTest.java

```diff
@@ -186,6 +186,7 @@ public void testExecuteAndPostProcess() throws Exception {
         r = getShowPageResult(a);
         actualData = (InstructorStudentRecordsPageData) r.data;
         expectedProfile.modifiedDate = actualData.spa.modifiedDate;
+        expectedProfile.gender = actualData.spa.gender;
```

Comment:
Why is this necessary?

Reply:
Sir here we have to gender of the expected profile to make it match with the actual profile as it is picking up the attributes so to remove in consistency overwriting is necessary

Reply:
Sorry, I'm not getting what you're trying to say. What kind of inconsistency?